src/test/user/register/post.controller.test.js

```diff
@@ -61,18 +63,35 @@ describe('User Register Post Controller', () => {
   it('should fail validation on erroneous submit', async () => {
     const emptyRequest = {
       body: {
-        Username: '',
+        userFname: '',
+        userLname: '',
+        userId: '',
+        cUserId: '',
       },
       session: {
         cookie: {},
       },
     };
-    try {
+    const cookie = new CookieModel(emptyRequest);
+
+    callController = async () => {
       await controller(emptyRequest, res);
-    } catch (err) {
-      expect(err).to.eq('Validation error when logging in');
-      expect(res.render).to.have.been.calledWith('app/user/login/index');
```

Comment:
Made the assertion here much stronger, hence the sudden addition of exact parameters.